src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
 public class Duke {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+    
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greet();
+
+        String userResponde;
+        Scanner in = new Scanner(System.in);
+        do {
+            userResponde = in.nextLine();
+            dukeResponde(userResponde);
+
+        }while(!userResponde.equals("bye"));
+
     }
+
+    public static void greet(){
+        printLine();
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        printLine();
+        System.out.println();
+    }
+
+    public static void dukeResponde(String userResponde){
+        printLine();
+        if(userResponde.equals("bye")){
+            sayBye();
+        }
+        else if(userResponde.equals("list")) {
+            listTask();
+        }
+        else if(userResponde.startsWith("done")) {
+            markAsDone(userResponde);
+        }
+        else{
+            addNewTask(userResponde);
+        }
```

Comment:
With reference to the coding standard, it is recommended that "else" is on the same line as "}" of the previous case.
if (xxx) {
} else {
}